packages/url/package.json

```diff
@@ -61,6 +61,7 @@
   "ava": {
     "files": [
       "!**/fixtures/**",
+      "!**/output/**",
```

Comment:
why was this necessary? I've not seen an issue with `output` in any of my tests

Reply:
Each test run left the final `/output` behind from the last unit that ran, and ava detected it the next time I ran all tests:

>   ✖ No tests found in test/output/bundle.js, make sure to import "ava" at the top of your test file

Could also solve by changing beforeEach to afterEach, but I guessed you had good reasons for using beforeEach.

Reply:
Weird. I never had that issue last night.